src/index.html

```diff
@@ -1,17 +1,164 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <div class="form_fields">
+          <label for="surname">Surname:</label>
+          <input
+            required
+            type="text"
+            name="surname"
+            id="surname"
+            minlength="2"
+            maxlength="30"
+          >
+        </div>
+        <div class="form_fields">
+          <label for="name">Name:</label>
+          <input type="text" name="name" id="name" >
+        </div>
+        <div class="form_fields">
+          <label for="age">How old are You? </label>
```

Comment:
Лучше выносить текст на отдельную строку
```suggestion
          <label for="age">
            How old are You?
          </label>
```